sdcm/cluster.py

```diff
@@ -3211,7 +3214,7 @@ class BaseCluster:  # pylint: disable=too-many-instance-attributes,too-many-publ
 
     # pylint: disable=too-many-arguments,too-many-locals,too-many-branches
     def __init__(self, cluster_uuid=None, cluster_prefix='cluster', node_prefix='node', n_nodes=3, params=None,
-                 region_names=None, node_type=None, extra_network_interface=False, add_nodes=True):
+                 region_names=None, node_type=None, extra_network_interface=False, add_nodes=True, nodes_smp=[]):
```

Comment:
don't initialize an argument with a mutable list
set it as None, and later in code initialize the list if needed